llvm/test/Transforms/LoopVectorize/tail-folding-optimize-vector-induction-width.ll

```diff
@@ -0,0 +1,1006 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -p loop-vectorize -force-vector-width=8 -prefer-predicate-over-epilogue=predicate-dont-vectorize -S %s | FileCheck %s
```

Comment:
Do we need VF=8 here or could we also choose 2, to get more compact IR?

Reply:
Ah good point - done.